src/types/donation.ts

```diff
@@ -43,7 +44,12 @@ export const zDonationEntity = zDonationBase.extend({ ...zBase.shape });
 
 export const zCreateDonationRequest = zDonationBase;
 
-export const zDonationResponse = zDonationEntity;
+//export const zDonationResponse = zDonationEntity;
+export const zDonationResponse = zDonationEntity.extend({
+  user: zUserResponse,
+  donor: zDonorResponse,
+  items: zDonationItemResponse,
```

Comment:
can you try to do `items: zDonationItemResponse[],`

Reply:
Gotcha.

That's giving me a syntax error:

 Expression expected
    ,-[/Users/tyanderson/Desktop/H4I/mission-of-hope/src/types/donation.ts:48:1]
 48 | export const zDonationResponse = zDonationEntity.extend({
 49 |   user: zUserResponse,
 50 |   donor: zDonorResponse,
 51 |   items: zDonationItemResponse[],
    :                                ^
 52 | });
 53 | 
 54 | export interface DonationEntity extends z.infer<typeof zDonationEntity> {}
    `----

Caused by:
    Syntax Error